### PR TITLE
chore: 🔧 fix Semantic Release version on CI/CD 

### PR DESCRIPTION
**Pull request by Gwenevere**
# Pull Request

## 📖 Description

The semantic-release version wasn't compatible with the semantic-release-monorepo and a "release-fallback" branch was neccessary in the configuration.

### 🎫 Issues

* The release was not successful due to the issues described above.

## 👩‍💻 Reviewer Notes

These were the results from the previous release that failed, so you can see the errors: https://github.com/SickDesignSystem/synergy/actions/runs/6585242049/job/17891283869 
This PR is the only way to test this functionality.

### General

- [ ] I have added tests for my changes.
- [ ] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have made sure to follow the projects coding and contribution guides.

### CODE REVIEW

File: packages/assets/package.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
     "eslint": "^8.47.0",
     "figma-export-assets": "github:mariohamann/figma-export-assets",
     "rimraf": "^5.0.1",
-    "semantic-release": "^22.0.5",
+    "semantic-release": "^19.0.5",
```
Comment (schilchSICKAG) [inline packages/assets/package.json:18]: What is the reason for the downgrade?

Comment (Gwenevere) [reply packages/assets/package.json:18]: the semantic-release version we are using is not compatible with the semantic-release-monorepo current version. At least I think that is the problem

Comment (schilchSICKAG) [reply packages/assets/package.json:18]: Ok, so we should keep an eye on that

Comment (mariohamann) [reply packages/assets/package.json:18]: See here: https://github.com/pmowrer/semantic-release-monorepo/issues/138

**Review by mariohamann — APPROVED** [review]
LGTM.

**Review by schilchSICKAG — APPROVED** [review]

**Pull request merged**